src/compiler/Fork.js

```diff
@@ -42,7 +42,7 @@ module.exports = class Fork extends EventEmitter {
       // WebSocket connection is established after the Fork is created.
       transportServer.on('connection', socket => {
         const platformMatch = socket.upgradeReq.url.match(
-          /platform=(ios|android)/
+          /platform=(.*)([&]|$)/
```

Comment:
We pass only one platform to the Fork. Maybe we can just base this regex on `platform` param? wdyt @zamotany?

Reply:
What's the reasoning behind this change??

Reply:
I see @acoates-ms wouldn't `([^&]*)` do the same job?

Reply:
Sure thing -- I've gone ahead and updated it.

Reply:
Alternatively we can make it more strict: `platform=([a-zA-z]+)`

Reply:
I don't think we should be that opinionated about what a valid platform name can be.  Ex: should '-' be a valid platform name character?  '_'?